### DISCUSSION TITLE
Current state of the new stuff from Terraria 1.4

### DISCUSSION THREAD

**Discussion by Syshein** [Pryaxis/TShock#2149] [Q&A] [answered]
Hey guys,

first of all: You are doing a great job :)

I asked this question a few days ago in the "Yes we are still alive" Thread but since it didn't get any replys (but newer discussions got) i hope it's ok to ask this in a new discussion.

I know it's a taboo to ask for your activities on this repo but can you tell me something about the current state of development? A few friends of mine and I would like to play the new content of Terraria 1.4(.1) and I need to host a server for them. But i can't find any info wether all new items, bosses and stuff are supported by the current tshock version. It would be really frustrating for all of us if I set up a server just to find that the new stuff is still unsupported.

I really don't want to rush anyone so no matter if the new things are working already or not. thanks for all the great work you are doing.

**Comment by EMREOYUN** [comment, marked as answer]
Currently "as a mobile server owner", bouncer has issues. Banned projectiles can be usable because of Bouncer bugs. All other things works very well. You can host a 1.4 server with your friend but not publicly.

**Reply by Syshein** [reply]
Just to make sure i got that right: The bouncer issue is only a problem when i want to ban some types of projectiles because it's not working right and the players can use the projectiles nevertheless?

**Reply by EMREOYUN** [reply]
I banned couple projectiles but players can use(by very low chance) that projectiles. I banned Empress Of Light's critter both item and projectile and i saw someone spawned it in front of me.
